base-notebook/start.sh

```diff
@@ -67,23 +68,23 @@ if [ $(id -u) == 0 ] ; then
     # Ex: default NFS/EFS (no auto-uid/gid)
     if [[ "$CHOWN_HOME" == "1" || "$CHOWN_HOME" == 'yes' ]]; then
         echo "Changing ownership of /home/$NB_USER to $NB_UID:$NB_GID with options '${CHOWN_HOME_OPTS}'"
-        chown $CHOWN_HOME_OPTS $NB_UID:$NB_GID /home/$NB_USER
+        chown "$CHOWN_HOME_OPTS" "$NB_UID:$NB_GID" "/home/$NB_USER"
```

Comment:
Hmmm, if for example CHOWN_HOME_OPTS are "--flag1 --flag2" then it is wrong to add quotes here I think. Note sure what CHOWN_HOME_OPTS may be though.

Reply:
Yes, I think you're right. Changed the code and added `shellcheck disable`.